_posts/2018-05-24-mentorship-open-source-hackillinois.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Open Source at HackIllinois 2018"
+date: 2018-05-24 00:00:00
+categories: recap
+tags: HackIllinois2018, featured
+image: /assets/article_images/mentors-open-source-hackillinois/panel.jpg
+author: Rohin Bhasin
+---
+
+Open Source carried a huge presence at HackIllinois 2018. This year, we had over [40 mentors](https://2018.hackillinois.org/mentors) and 25 sponsors at HackIllinois, and everyone came eager to contribute to the open source community! The goal of HackIllinois has transitioned in recent years to connect with the developer community; to encourage students and hackers from all over the country to contribute to the tools they use every day. HackIllinois 2018 was a giant step towards fulfilling this vision.
+
+# Mentors
+
+Open source contributions at HackIllinois would not be possible without the many amazing mentors that attended the event this year. Mentorship is essential to the HackIllinois experience; these experienced open source contributors and engineers help students get the ball rolling on contributing to open source. 
```

Comment:
In the second sentence, the first clause introduces the second: consider using a colon (:) instead of a semicolon (;).